### DISCUSSION TITLE
多模网关无实体

### DISCUSSION THREAD

**Discussion by zraxx** [XiaoMi/ha_xiaomi_home#402] [Q&A]
![Screenshot_2024-12-24-00-17-13-850_io.homeassistant.companion.android.png](https://github.com/user-attachments/assets/9949cdae-1a71-4c97-91e6-612c300c4da0)

**Comment by huhuime** [comment]
更新固件试试，我这1.5.6_0001是正常的
![image](https://github.com/user-attachments/assets/7a9db955-14fb-46f8-86e2-9c96029471a8)

**Comment by ooodwbooo** [comment]
ha要是能通过多模网关离线控制就好了，纠结要不要买中枢网关。

**Reply by refined-fish** [reply]
没有mqtt就不可能呀，所以别想了😂
